oscarapi/apps/basket/abstract_models.py

```diff
@@ -0,0 +1,101 @@
+from django.conf import settings
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from oscar.apps.basket.abstract_models import AbstractBasket as _AbstractBasket, AbstractLine as _AbstractLine
+from oscar.apps.basket.managers import OpenBasketManager, SavedBasketManager
+from oscar.core.utils import get_default_currency
+
+from oscarapi.apps.basket.managers import EditableBasketManager
+from oscarapi.apps.basket.utils import get_basket
+
+
+__all__ = ('AbstractBasket', 'AbstractLine')
+
+
+class AbstractBasket(_AbstractBasket):
```

Comment:
Another option might be to avoid overriding the basket in this package but leave that to your specific project (so everyone isn't forced to use the overrides). Not sure if that would suit how things are structured though.